src/lazy/binary/raw/v1_1/value.rs

```diff
@@ -266,7 +255,7 @@ impl<'top> LazyRawBinaryValue_1_1<'top> {
             (OpcodeType::Integer, 0x0) => 0.into(),
             (OpcodeType::Integer, n) => {
                 // We have n bytes following that make up our integer.
-                self.input.consume(1).read_fixed_int(n)?.0.into()
+                self.available_body().read_fixed_int(n)?.0.into()
```

Comment:
🗺️ Adding annotations support meant changing a few usages of `self.input` because that buffer contains the annotations sequence (previously empty) and the value itself.